apps/workshop/projects/CroSwap/modules/CroSwap.ts

```diff
@@ -72,42 +73,37 @@ export const CroSwap: ModuleDefinitionInterface = {
       // total balance of lp tokens in wallet & staked
       const totalBalance = balance.add(userInfo.amount);
 
-      // dont continue if no balance
-      if (totalBalance.eq(0)) return void 0;
-
-      const reserve0 = ethers.FixedNumber.fromValue(reserves._reserve0, 18);
-      const reserve1 = ethers.FixedNumber.fromValue(reserves._reserve1, 18);
-
-      // get ratio of total supply
-      const ratioOfTotalSupply = ethers.FixedNumber.fromValue(totalBalance).divUnsafe(
-        ethers.FixedNumber.from(totalSupply),
-      );
-
-      const token0Balance = parseFloat(
-        ethers.utils.formatUnits(
-          reserve0.mulUnsafe(ratioOfTotalSupply).toHexString(),
-          token0.decimals,
-        ),
-      );
-      const token1Balance = parseFloat(
-        ethers.utils.formatUnits(
-          reserve1.mulUnsafe(ratioOfTotalSupply).toHexString(),
-          token1.decimals,
-        ),
-      );
+      // // dont continue if no balance
```

Comment:
Please delete all not necessary comments

Reply:
removed